### PR TITLE
📝  Release eds-core-react and eds-tokens

### LINKED ISSUE #1140: Prepare next release
Shortlist of features in next release:

## core-react

* Textfield improvements

## tokens

- css-variable with fallback

### PR DESCRIPTION
**Pull request by wenche**
Resolve #1140

### CODE REVIEW

**Comment by wenche** [comment]
Suggestions for improvements @mimarz ?

**Comment by wenche** [comment]
I'm not 100% sure about which date to add.

**Review by pomfrida — APPROVED** [review]
LGTM 👍

File: libraries/core-react/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.0] - 2021-03-x
```
Comment (pomfrida) [inline libraries/core-react/CHANGELOG.md:8]:
```suggestion
## [0.9.0] - 2021-03-17
```

File: libraries/tokens/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.6] - 2021-02-x
```
Comment (pomfrida) [inline libraries/tokens/CHANGELOG.md:8]:
```suggestion
## [0.5.6] - 2021-03-17
```

**Review by pomfrida — CHANGES_REQUESTED** [review]
Forgot to add dates

**Review by pomfrida — APPROVED** [review]
Now it's good to go 👍

**Pull request merged**